src/components/CreationMapView/index.ts

```diff
@@ -300,8 +301,10 @@ export default class CreationMapView extends Vue {
 
     /**
      * マップクリック時に実行される関数を，形状描画メソッドにする
+     * @param onShapeCreated 形状の作成が完了したときに呼び出す関数
      */
-    private setAddPointMethodOnMapClick(): void {
+    private setAddPointMethodOnMapClick(onShapeCreated: () => void): void {
```

Comment:
```suggestion
    private setAddPointMethodOnMapClick(onShapeCreated: () => void = () => {}): void {
```
引数を渡さない場合にエラーが起きてしまうので、デフォルト値を設定してみるのはどうでしょうか

Reply:
空ブロックはundefindにするとワーニング回避できます

Reply:
なるほど！